services/src/services/errorTelemetry/errorTelemetry.service.js

```diff
@@ -7,16 +7,34 @@
  * simply be taking the last of version records associated with the instance.
  */
 
-const createService = require('feathers-sequelize');
-const createModel = require('../../models/error_log');
+const fs             = require('fs');
+
 const hooks = require('./errorTelemetry.hooks');
 
 module.exports = function (app) {
-  const options = {
-    name: 'error_log',
-    Model: createModel(app)
-  };
+  class MyService {
```

Comment:
This class should not be called `MyService` :smile: 

Also, look at the Authentication service for a pattern for factoring the class in a separate file and then incorporating it into the service module. That service follows the pattern feathersjs' CLI will generate for a non-persistence backed service.